source/includes/changelog/_19-04-0.md.erb

```diff
@@ -7,3 +7,4 @@
 ### Removed
 
 ### Deprecations
+- The Plugin Info API version 5 has been deprecated. This version will be removed in a release scheduled for August 2019. Version 6 of the API is available, and users are encouraged to use it.
```

Comment:
Add a note in additions for the new API version?

Reply:
done